corebuild/integration/ILLink.CustomSteps/ClearInitLocals.cs

```diff
@@ -8,17 +9,39 @@ namespace ILLink.CustomSteps
 {
 	public class ClearInitLocalsStep : BaseStep
 	{
-		protected override void ProcessAssembly(AssemblyDefinition assembly)
+		HashSet<string> _assemblies;
+
+		protected override void Process ()
+		{
+			string parameterName = "ClearInitLocalsAssemblies";
+
+			if (Context.HasParameter (parameterName)) {
+				string parameter = Context.GetParameter (parameterName);
+				_assemblies = new HashSet<string> (parameter.Split(','), StringComparer.OrdinalIgnoreCase);
+			}
+		}
+
+		protected override void ProcessAssembly (AssemblyDefinition assembly)
 		{
+			if ((_assemblies != null) && (!_assemblies.Contains (assembly.Name.Name))) {
+				return;
+			}
+
+			bool changed = false;
+
 			foreach (ModuleDefinition module in assembly.Modules) {
 				foreach (TypeDefinition type in module.Types) {
 					foreach (MethodDefinition method in type.Methods) {
 						if (method.Body != null) {
 							method.Body.InitLocals = false;
+							changed = true;
```

Comment:
Maybe check whether InitLocals is already false before setting changed to true? Not sure how likely we are to encounter an assembly that already has this set to false for all methods, so it may not make much of a difference. It's your call.

Reply:
Done.